### PR TITLE
A solution for device tree (dt)

### PR DESCRIPTION
**Pull request by malokhvii-eduard**
## :pushpin: Description

Some device tree configurations for the I2C adapter №0 and [Bosch Sensortec BME280 sensor](https://www.bosch-sensortec.com/products/environmental-sensors/humidity-sensors-bme280/). See [sources of I2C driver for Bosch Sensortec BME280 sensor](https://gitlab.com/malokhvii-eduard/bme280-driver).

## :page_with_curl: Booting process

```console
U-Boot SPL 2019.04-armbian (Nov 18 2019 - 23:02:49 +0100)
DRAM: 512 MiB
Trying to boot from MMC1


U-Boot 2019.04-armbian (Nov 18 2019 - 23:02:49 +0100) Allwinner Technology

CPU:   Allwinner H3 (SUN8I 1680)
Model: Xunlong Orange Pi One
DRAM:  512 MiB
MMC:   mmc@1c0f000: 0
Loading Environment from EXT4... ** File not found /boot/boot.env **

** Unable to read "/boot/boot.env" from mmc0:1 **
In:    serial
Out:   serial
Err:   serial
Net:   phy interface0
eth0: ethernet@1c30000
** Reading file would overwrite reserved memory **
There is no valid bmp file at the given address
starting USB...
USB0:   USB EHCI 1.00
USB1:   USB OHCI 1.0
USB2:   USB EHCI 1.00
USB3:   USB OHCI 1.0
scanning bus 0 for devices... 1 USB Device(s) found
scanning bus 1 for devices... 1 USB Device(s) found
scanning bus 2 for devices... 1 USB Device(s) found
scanning bus 3 for devices... 1 USB Device(s) found
       scanning usb for storage devices... 0 Storage Device(s) found
Autoboot in 1 seconds, press <Space> to stop
switch to partitions #0, OK
mmc0 is current device
Scanning mmc 0:1...
Found U-Boot script /boot/boot.scr
3798 bytes read in 2 ms (1.8 MiB/s)
## Executing script at 43100000
U-boot loaded from SD
Boot script loaded from mmc
222 bytes read in 1 ms (216.8 KiB/s)
7307841 bytes read in 352 ms (19.8 MiB/s)
7644632 bytes read in 369 ms (19.8 MiB/s)
Found mainline kernel configuration
28283 bytes read in 9 ms (3 MiB/s)
374 bytes read in 3 ms (121.1 KiB/s)
Applying kernel provided DT overlay sun8i-h3-i2c0.dtbo
508 bytes read in 5 ms (98.6 KiB/s)
Applying kernel provided DT overlay sun8i-h3-bme280.dtbo
4155 bytes read in 4 ms (1013.7 KiB/s)
Applying kernel provided DT fixup script (sun8i-h3-fixup.scr)
## Executing script at 44000000
## Loading init Ramdisk from Legacy Image at 43300000 ...
   Image Name:   uInitrd
   Image Type:   ARM Linux RAMDisk Image (gzip compressed)
   Data Size:    7307777 Bytes = 7 MiB
   Load Address: 00000000
   Entry Point:  00000000
   Verifying Checksum ... OK
## Flattened Device Tree blob at 43000000
   Booting using the fdt blob at 0x43000000
EHCI failed to shut down host controller.
   Loading Ramdisk to 49907000, end 49fff201 ... OK
   Loading Device Tree to 49897000, end 49906fff ... OK

Starting kernel ...

Uncompressing Linux... done, booting the kernel.
```

## :page_with_curl: Output of dmesg

```console
$ dmesg
[    0.000000] Booting Linux on physical CPU 0x0
[    0.000000] Linux version 5.3.9-sunxi (root@builder) (gcc version 8.3.0 (GNU Toolchain for the A-profile Architecture 8.3-2019.03 (arm-rel-8.36))) #19.11.3 SMP Mon Nov 18 18:49:43 CET 2019
[    0.000000] CPU: ARMv7 Processor [410fc075] revision 5 (ARMv7), cr=50c5387d
[    0.000000] CPU: div instructions available: patching division code
[    0.000000] CPU: PIPT / VIPT nonaliasing data cache, VIPT aliasing instruction cache
[    0.000000] OF: fdt: Machine model: Xunlong Orange Pi One
[    0.000000] Memory policy: Data cache writealloc
[    0.000000] cma: Reserved 128 MiB at 0x55c00000
[    0.000000] On node 0 totalpages: 131072
[    0.000000]   Normal zone: 1152 pages used for memmap
[    0.000000]   Normal zone: 0 pages reserved
[    0.000000]   Normal zone: 131072 pages, LIFO batch:31
[    0.000000] psci: probing for conduit method from DT.
[    0.000000] psci: Using PSCI v0.1 Function IDs from DT
[    0.000000] percpu: Embedded 20 pages/cpu s49676 r8192 d24052 u81920
[    0.000000] pcpu-alloc: s49676 r8192 d24052 u81920 alloc=20*4096
[    0.000000] pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3 
[    0.000000] Built 1 zonelists, mobility grouping on.  Total pages: 129920
[    0.000000] Kernel command line: root=UUID=24819d8b-5022-4647-a64d-eb65eea3ea11 rootwait rootfstype=ext4 console=ttyS0,115200 console=tty1 hdmi.audio=EDID:0 disp.screen0_output_mode=1920x1080p60 panic=10 consoleblank=0 loglevel=1 ubootpart=7d2a55ce-01 ubootsource=mmc usb-storage.quirks=0x2537:0x1066:u,0x2537:0x1068:u   sunxi_ve_mem_reserve=0 sunxi_g2d_mem_reserve=0 sunxi_fb_mem_reserve=16 cgroup_enable=memory swapaccount=1
[    0.000000] Dentry cache hash table entries: 65536 (order: 6, 262144 bytes, linear)
[    0.000000] Inode-cache hash table entries: 32768 (order: 5, 131072 bytes, linear)
[    0.000000] allocated 524288 bytes of page_ext
[    0.000000] mem auto-init: stack:off, heap alloc:off, heap free:off
[    0.000000] Memory: 363696K/524288K available (9216K kernel code, 1040K rwdata, 2772K rodata, 1024K init, 316K bss, 29520K reserved, 131072K cma-reserved, 0K highmem)
[    0.000000] random: get_random_u32 called from __kmem_cache_create+0x2b/0x3ac with crng_init=0
[    0.000000] SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
[    0.000000] ftrace: allocating 40210 entries in 79 pages
[    0.000000] rcu: Hierarchical RCU implementation.
[    0.000000] rcu: 	RCU restricting CPUs from NR_CPUS=8 to nr_cpu_ids=4.
[    0.000000] rcu: RCU calculated value of scheduler-enlistment delay is 25 jiffies.
[    0.000000] rcu: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
[    0.000000] NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
[    0.000000] GIC: Using split EOI/Deactivate mode
[    0.000000] clocksource: timer: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 79635851949 ns
[    0.000000] arch_timer: cp15 timer(s) running at 24.00MHz (phys).
[    0.000000] clocksource: arch_sys_counter: mask: 0xffffffffffffff max_cycles: 0x588fe9dc0, max_idle_ns: 440795202592 ns
[    0.000006] sched_clock: 56 bits at 24MHz, resolution 41ns, wraps every 4398046511097ns
[    0.000014] Switching to timer-based delay loop, resolution 41ns
[    0.000182] Console: colour dummy device 80x30
[    0.000194] printk: console [tty1] enabled
[    0.000237] Calibrating delay loop (skipped), value calculated using timer frequency.. 48.00 BogoMIPS (lpj=96000)
[    0.000247] pid_max: default: 32768 minimum: 301
[    0.000486] Mount-cache hash table entries: 1024 (order: 0, 4096 bytes, linear)
[    0.000495] Mountpoint-cache hash table entries: 1024 (order: 0, 4096 bytes, linear)
[    0.001157] *** VALIDATE proc ***
[    0.001415] *** VALIDATE cgroup1 ***
[    0.001422] *** VALIDATE cgroup2 ***
[    0.001476] CPU: Testing write buffer coherency: ok
[    0.001843] CPU0: thread -1, cpu 0, socket 0, mpidr 80000000
[    0.002310] Setting up static identity map for 0x40100000 - 0x40100054
[    0.002413] rcu: Hierarchical SRCU implementation.
[    0.002908] smp: Bringing up secondary CPUs ...
[    0.013589] CPU1: thread -1, cpu 1, socket 0, mpidr 80000001
[    0.024380] CPU2: thread -1, cpu 2, socket 0, mpidr 80000002
[    0.035097] CPU3: thread -1, cpu 3, socket 0, mpidr 80000003
[    0.035189] smp: Brought up 1 node, 4 CPUs
[    0.035197] SMP: Total of 4 processors activated (192.00 BogoMIPS).
[    0.035200] CPU: All CPU(s) started in HYP mode.
[    0.035202] CPU: Virtualization extensions available.
[    0.036164] devtmpfs: initialized
[    0.042185] VFP support v0.3: implementor 41 architecture 2 part 30 variant 7 rev 5
[    0.042407] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645041785100000 ns
[    0.042424] futex hash table entries: 1024 (order: 4, 65536 bytes, linear)
[    0.046983] xor: measuring software checksum speed
[    0.087032]    arm4regs  :  1255.000 MB/sec
[    0.127083]    8regs     :   874.000 MB/sec
[    0.167140]    32regs    :   826.000 MB/sec
[    0.207192]    neon      :  1271.000 MB/sec
[    0.207196] xor: using function: neon (1271.000 MB/sec)
[    0.207258] pinctrl core: initialized pinctrl subsystem
[    0.208232] NET: Registered protocol family 16
[    0.209905] DMA: preallocated 256 KiB pool for atomic coherent allocations
[    0.210425] audit: initializing netlink subsys (disabled)
[    0.210620] audit: type=2000 audit(0.184:1): state=initialized audit_enabled=0 res=1
[    0.211040] cpuidle: using governor ladder
[    0.211070] cpuidle: using governor menu
[    0.211591] hw-breakpoint: found 5 (+1 reserved) breakpoint and 4 watchpoint registers.
[    0.211595] hw-breakpoint: maximum watchpoint size is 8 bytes.
[    0.303447] raid6: neonx8   gen()   709 MB/s
[    0.371518] raid6: neonx8   xor()   590 MB/s
[    0.439594] raid6: neonx4   gen()   727 MB/s
[    0.507699] raid6: neonx4   xor()   616 MB/s
[    0.575771] raid6: neonx2   gen()   663 MB/s
[    0.643809] raid6: neonx2   xor()   584 MB/s
[    0.711977] raid6: neonx1   gen()   495 MB/s
[    0.780033] raid6: neonx1   xor()   467 MB/s
[    0.848084] raid6: int32x8  gen()   267 MB/s
[    0.916221] raid6: int32x8  xor()   174 MB/s
[    0.984242] raid6: int32x4  gen()   285 MB/s
[    1.052330] raid6: int32x4  xor()   195 MB/s
[    1.120564] raid6: int32x2  gen()   255 MB/s
[    1.188504] raid6: int32x2  xor()   206 MB/s
[    1.256748] raid6: int32x1  gen()   200 MB/s
[    1.324812] raid6: int32x1  xor()   176 MB/s
[    1.324816] raid6: using algorithm neonx4 gen() 727 MB/s
[    1.324819] raid6: .... xor() 616 MB/s, rmw enabled
[    1.324823] raid6: using neon recovery algorithm
[    1.326403] SCSI subsystem initialized
[    1.326623] libata version 3.00 loaded.
[    1.326828] usbcore: registered new interface driver usbfs
[    1.326868] usbcore: registered new interface driver hub
[    1.326944] usbcore: registered new device driver usb
[    1.327111] mc: Linux media interface: v0.10
[    1.327138] videodev: Linux video capture interface: v2.00
[    1.327281] pps_core: LinuxPPS API ver. 1 registered
[    1.327285] pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
[    1.327300] PTP clock support registered
[    1.328746] clocksource: Switched to clocksource arch_sys_counter
[    2.149005] VFS: Disk quotas dquot_6.6.0
[    2.149086] VFS: Dquot-cache hash table entries: 1024 (order 0, 4096 bytes)
[    2.156432] thermal_sys: Registered thermal governor 'fair_share'
[    2.156438] thermal_sys: Registered thermal governor 'bang_bang'
[    2.156444] thermal_sys: Registered thermal governor 'step_wise'
[    2.156447] thermal_sys: Registered thermal governor 'power_allocator'
[    2.156951] NET: Registered protocol family 2
[    2.157582] tcp_listen_portaddr_hash hash table entries: 512 (order: 0, 6144 bytes, linear)
[    2.157610] TCP established hash table entries: 4096 (order: 2, 16384 bytes, linear)
[    2.157649] TCP bind hash table entries: 4096 (order: 3, 32768 bytes, linear)
[    2.157707] TCP: Hash tables configured (established 4096 bind 4096)
[    2.157798] UDP hash table entries: 256 (order: 1, 8192 bytes, linear)
[    2.157831] UDP-Lite hash table entries: 256 (order: 1, 8192 bytes, linear)
[    2.158021] NET: Registered protocol family 1
[    2.158498] RPC: Registered named UNIX socket transport module.
[    2.158503] RPC: Registered udp transport module.
[    2.158507] RPC: Registered tcp transport module.
[    2.158510] RPC: Registered tcp NFSv4.1 backchannel transport module.
[    2.158724] Trying to unpack rootfs image as initramfs...
[    2.588353] Freeing initrd memory: 7140K
[    2.590147] Initialise system trusted keyrings
[    2.590381] workingset: timestamp_bits=14 max_order=17 bucket_order=3
[    2.596587] zbud: loaded
[    2.598905] NFS: Registering the id_resolver key type
[    2.598936] Key type id_resolver registered
[    2.598939] Key type id_legacy registered
[    2.598951] nfs4filelayout_init: NFSv4 File Layout Driver Registering...
[    2.598956] Installing knfsd (copyright (C) 1996 okir@monad.swb.de).
[    2.600308] JFS: nTxBlock = 3921, nTxLock = 31369
[    2.604994] SGI XFS with ACLs, security attributes, realtime, no debug enabled
[    2.656422] Key type asymmetric registered
[    2.656431] Asymmetric key parser 'x509' registered
[    2.656502] Block layer SCSI generic (bsg) driver version 0.4 loaded (major 246)
[    2.656701] io scheduler mq-deadline registered
[    2.656707] io scheduler kyber registered
[    2.656944] io scheduler bfq registered
[    2.661870] sun8i-h3-pinctrl 1c20800.pinctrl: initialized sunXi PIO driver
[    2.663346] sun8i-h3-r-pinctrl 1f02c00.pinctrl: initialized sunXi PIO driver
[    2.668897] Serial: 8250/16550 driver, 8 ports, IRQ sharing disabled
[    2.670385] sun8i-h3-pinctrl 1c20800.pinctrl: 1c20800.pinctrl supply vcc-pa not found, using dummy regulator
[    2.670774] printk: console [ttyS0] disabled
[    2.690951] 1c28000.serial: ttyS0 at MMIO 0x1c28000 (irq = 36, base_baud = 1500000) is a U6_16550A
[    2.691003] printk: console [ttyS0] enabled
[    2.705560] sun4i-drm display-engine: bound 1100000.mixer (ops 0xc0a8c638)
[    2.705826] sun4i-drm display-engine: bound 1c0c000.lcd-controller (ops 0xc0a89220)
[    2.705890] sun8i-dw-hdmi 1ee0000.hdmi: 1ee0000.hdmi supply hvcc not found, using dummy regulator
[    2.706763] sun8i-dw-hdmi 1ee0000.hdmi: Detected HDMI TX controller v1.32a with HDCP (sun8i_dw_hdmi_phy)
[    2.707153] sun8i-dw-hdmi 1ee0000.hdmi: registered DesignWare HDMI I2C bus driver
[    2.707395] sun4i-drm display-engine: bound 1ee0000.hdmi (ops 0xc0a8bc24)
[    2.707401] [drm] Supports vblank timestamp caching Rev 2 (21.10.2013).
[    2.707405] [drm] No driver support for vblank timestamp query.
[    2.707755] [drm] Initialized sun4i-drm 1.0.0 20150629 for display-engine on minor 0
[    2.707826] [drm] Cannot find any crtc or sizes
[    2.708553] [drm] Cannot find any crtc or sizes
[    2.710929] brd: module loaded
[    2.716366] loop: module loaded
[    2.718356] libphy: Fixed MDIO Bus: probed
[    2.718842] dwmac-sun8i 1c30000.ethernet: PTP uses main clock
[    2.718887] dwmac-sun8i 1c30000.ethernet: 1c30000.ethernet supply phy not found, using dummy regulator
[    2.718959] dwmac-sun8i 1c30000.ethernet: 1c30000.ethernet supply phy-io not found, using dummy regulator
[    2.719297] dwmac-sun8i 1c30000.ethernet: Current syscon value is not the default 148000 (expect 58000)
[    2.719317] dwmac-sun8i 1c30000.ethernet: No HW DMA feature register supported
[    2.719323] dwmac-sun8i 1c30000.ethernet: RX Checksum Offload Engine supported
[    2.719329] dwmac-sun8i 1c30000.ethernet: COE Type 2
[    2.719335] dwmac-sun8i 1c30000.ethernet: TX Checksum insertion supported
[    2.719341] dwmac-sun8i 1c30000.ethernet: Normal descriptors
[    2.719346] dwmac-sun8i 1c30000.ethernet: Chain mode enabled
[    2.719483] libphy: stmmac: probed
[    2.719940] dwmac-sun8i 1c30000.ethernet: Found internal PHY node
[    2.720040] libphy: mdio_mux: probed
[    2.720058] dwmac-sun8i 1c30000.ethernet: Switch mux to internal PHY
[    2.720066] dwmac-sun8i 1c30000.ethernet: Powering internal PHY
[    2.730735] libphy: mdio_mux: probed
[    2.731586] ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
[    2.731591] ehci-platform: EHCI generic platform driver
[    2.731778] ehci-platform 1c1a000.usb: EHCI Host Controller
[    2.731805] ehci-platform 1c1a000.usb: new USB bus registered, assigned bus number 1
[    2.732206] ehci-platform 1c1a000.usb: irq 26, io mem 0x01c1a000
[    2.744748] ehci-platform 1c1a000.usb: USB 2.0 started, EHCI 1.00
[    2.745011] usb usb1: New USB device found, idVendor=1d6b, idProduct=0002, bcdDevice= 5.03
[    2.745019] usb usb1: New USB device strings: Mfr=3, Product=2, SerialNumber=1
[    2.745025] usb usb1: Product: EHCI Host Controller
[    2.745031] usb usb1: Manufacturer: Linux 5.3.9-sunxi ehci_hcd
[    2.745037] usb usb1: SerialNumber: 1c1a000.usb
[    2.745567] hub 1-0:1.0: USB hub found
[    2.745615] hub 1-0:1.0: 1 port detected
[    2.746225] ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
[    2.746248] ohci-platform: OHCI generic platform driver
[    2.746401] ohci-platform 1c1a400.usb: Generic Platform OHCI controller
[    2.746422] ohci-platform 1c1a400.usb: new USB bus registered, assigned bus number 2
[    2.746613] ohci-platform 1c1a400.usb: irq 27, io mem 0x01c1a400
[    2.808989] usb usb2: New USB device found, idVendor=1d6b, idProduct=0001, bcdDevice= 5.03
[    2.808998] usb usb2: New USB device strings: Mfr=3, Product=2, SerialNumber=1
[    2.809004] usb usb2: Product: Generic Platform OHCI controller
[    2.809010] usb usb2: Manufacturer: Linux 5.3.9-sunxi ohci_hcd
[    2.809016] usb usb2: SerialNumber: 1c1a400.usb
[    2.809500] hub 2-0:1.0: USB hub found
[    2.809548] hub 2-0:1.0: 1 port detected
[    2.810527] usbcore: registered new interface driver usb-storage
[    2.811362] sun6i-rtc 1f00000.rtc: registered as rtc0
[    2.811368] sun6i-rtc 1f00000.rtc: RTC enabled
[    2.811597] i2c /dev entries driver
[    2.813570] sunxi-wdt 1c20ca0.watchdog: Watchdog enabled (timeout=16 sec, nowayout=0)
[    2.814176] sun8i-h3-pinctrl 1c20800.pinctrl: 1c20800.pinctrl supply vcc-pf not found, using dummy regulator
[    2.814896] sunxi-mmc 1c0f000.mmc: Got CD GPIO
[    2.840197] sunxi-mmc 1c0f000.mmc: initialized, max. request size: 16384 KB
[    2.840627] sun8i-h3-r-pinctrl 1f02c00.pinctrl: 1f02c00.pinctrl supply vcc-pl not found, using dummy regulator
[    2.841280] ledtrig-cpu: registered to indicate activity on CPUs
[    2.841344] hidraw: raw HID events driver (C) Jiri Kosina
[    2.841462] usbcore: registered new interface driver usbhid
[    2.841465] usbhid: USB HID core driver
[    2.843711] Initializing XFRM netlink socket
[    2.844360] NET: Registered protocol family 10
[    2.876818] mmc0: host does not support reading read-only switch, assuming write-enable
[    2.879561] mmc0: new high speed SDHC card at address aaaa
[    2.880992] mmcblk0: mmc0:aaaa SC16G 14.8 GiB 
[    2.883808]  mmcblk0: p1
[    2.905310] Segment Routing with IPv6
[    2.905420] NET: Registered protocol family 17
[    2.905450] NET: Registered protocol family 15
[    2.905521] bridge: filtering via arp/ip/ip6tables is no longer available by default. Update your scripts to load br_netfilter if you need this.
[    2.905713] 8021q: 802.1Q VLAN Support v1.8
[    2.905776] Key type dns_resolver registered
[    2.906220] Registering SWP/SWPB emulation handler
[    2.906764] registered taskstats version 1
[    2.906768] Loading compiled-in X.509 certificates
[    2.906859] zswap: loaded using pool lzo/zbud
[    2.909439] Btrfs loaded, crc32c=crc32c-generic
[    2.938845] Key type encrypted registered
[    2.948332] sun8i-h3-pinctrl 1c20800.pinctrl: 1c20800.pinctrl supply vcc-pg not found, using dummy regulator
[    2.949666] ehci-platform 1c1b000.usb: EHCI Host Controller
[    2.949701] ehci-platform 1c1b000.usb: new USB bus registered, assigned bus number 3
[    2.950021] ehci-platform 1c1b000.usb: irq 28, io mem 0x01c1b000
[    2.964771] ehci-platform 1c1b000.usb: USB 2.0 started, EHCI 1.00
[    2.964977] usb usb3: New USB device found, idVendor=1d6b, idProduct=0002, bcdDevice= 5.03
[    2.964985] usb usb3: New USB device strings: Mfr=3, Product=2, SerialNumber=1
[    2.964992] usb usb3: Product: EHCI Host Controller
[    2.964998] usb usb3: Manufacturer: Linux 5.3.9-sunxi ehci_hcd
[    2.965004] usb usb3: SerialNumber: 1c1b000.usb
[    2.965546] hub 3-0:1.0: USB hub found
[    2.965583] hub 3-0:1.0: 1 port detected
[    2.966358] ohci-platform 1c1b400.usb: Generic Platform OHCI controller
[    2.966380] ohci-platform 1c1b400.usb: new USB bus registered, assigned bus number 4
[    2.966589] ohci-platform 1c1b400.usb: irq 29, io mem 0x01c1b400
[    3.028932] usb usb4: New USB device found, idVendor=1d6b, idProduct=0001, bcdDevice= 5.03
[    3.028940] usb usb4: New USB device strings: Mfr=3, Product=2, SerialNumber=1
[    3.028946] usb usb4: Product: Generic Platform OHCI controller
[    3.028952] usb usb4: Manufacturer: Linux 5.3.9-sunxi ohci_hcd
[    3.028958] usb usb4: SerialNumber: 1c1b400.usb
[    3.029408] hub 4-0:1.0: USB hub found
[    3.029452] hub 4-0:1.0: 1 port detected
[    3.030238] usb_phy_generic usb_phy_generic.2.auto: usb_phy_generic.2.auto supply vcc not found, using dummy regulator
[    3.030626] musb-hdrc musb-hdrc.3.auto: MUSB HDRC host driver
[    3.030637] musb-hdrc musb-hdrc.3.auto: new USB bus registered, assigned bus number 5
[    3.030843] usb usb5: New USB device found, idVendor=1d6b, idProduct=0002, bcdDevice= 5.03
[    3.030851] usb usb5: New USB device strings: Mfr=3, Product=2, SerialNumber=1
[    3.030858] usb usb5: Product: MUSB HDRC host driver
[    3.030864] usb usb5: Manufacturer: Linux 5.3.9-sunxi musb-hcd
[    3.030870] usb usb5: SerialNumber: musb-hdrc.3.auto
[    3.031305] hub 5-0:1.0: USB hub found
[    3.031341] hub 5-0:1.0: 1 port detected
[    3.032216] sun6i-rtc 1f00000.rtc: setting system clock to 2020-02-10T12:28:00 UTC (1581337680)
[    3.032399] of_cfs_init
[    3.032515] of_cfs_init: OK
[    3.035879] Freeing unused kernel memory: 1024K
[    3.044964] Run /init as init process
[    3.328827] input: r_gpio_keys as /devices/platform/r_gpio_keys/input/input0
[    3.849851] random: fast init done
[    3.869641] EXT4-fs (mmcblk0p1): mounted filesystem with writeback data mode. Opts: (null)
[    4.576465] systemd[1]: systemd 241 running in system mode. (+PAM +AUDIT +SELINUX +IMA +APPARMOR +SMACK +SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD -IDN2 +IDN -PCRE2 default-hierarchy=hybrid)
[    4.577161] systemd[1]: Detected architecture arm.
[    4.596322] systemd[1]: Set hostname to <chlorine>.
[    4.602407] systemd[1]: Failed to bump fs.file-max, ignoring: Invalid argument
[    5.213529] random: systemd: uninitialized urandom read (16 bytes read)
[    5.225264] random: systemd: uninitialized urandom read (16 bytes read)
[    5.225346] systemd[1]: Reached target System Time Synchronized.
[    5.226048] random: systemd: uninitialized urandom read (16 bytes read)
[    5.229052] systemd[1]: Created slice User and Session Slice.
[    5.230353] systemd[1]: Set up automount Arbitrary Executable File Formats File System Automount Point.
[    5.230743] systemd[1]: Reached target Remote File Systems.
[    5.231257] systemd[1]: Listening on initctl Compatibility Named Pipe.
[    5.232223] systemd[1]: Listening on Journal Audit Socket.
[    5.376960] bme280: loading out-of-tree module taints kernel.
[    5.396826] bme280: register device at i2c-0 0x76
[    5.413628] EXT4-fs (mmcblk0p1): re-mounted. Opts: commit=600,errors=remount-ro
[    5.774209] systemd-journald[279]: Received request to flush runtime journal from PID 1
[    6.000343] core: _opp_supported_by_regulators: OPP minuV: 1320000 maxuV: 1320000, not supported by regulator
[    6.000367] cpu cpu0: _opp_add: OPP not supported by regulators (1104000000)
[    6.000547] core: _opp_supported_by_regulators: OPP minuV: 1320000 maxuV: 1320000, not supported by regulator
[    6.000562] cpu cpu0: _opp_add: OPP not supported by regulators (1200000000)
[    6.000698] core: _opp_supported_by_regulators: OPP minuV: 1340000 maxuV: 1340000, not supported by regulator
[    6.000709] cpu cpu0: _opp_add: OPP not supported by regulators (1296000000)
[    6.000923] core: _opp_supported_by_regulators: OPP minuV: 1400000 maxuV: 1400000, not supported by regulator
[    6.000935] cpu cpu0: _opp_add: OPP not supported by regulators (1368000000)
[    6.271567] lima 1c40000.gpu: gp - mali400 version major 1 minor 1
[    6.271629] lima 1c40000.gpu: pp0 - mali400 version major 1 minor 1
[    6.271687] lima 1c40000.gpu: pp1 - mali400 version major 1 minor 1
[    6.271762] lima 1c40000.gpu: l2 cache 64K, 4-way, 64byte cache line, 64bit external bus
[    6.288615] lima 1c40000.gpu: bus rate = 200000000
[    6.288639] lima 1c40000.gpu: mod rate = 384000000
[    6.296034] [drm] Initialized lima 1.0.0 20190217 for 1c40000.gpu on minor 1
[    6.437145] zram: Added device: zram0
[    6.439391] zram: Added device: zram1
[    6.443724] zram: Added device: zram2
[    6.551471] zram1: detected capacity change from 0 to 257503232
[    6.632925] asoc-simple-card soc:sound: i2s-hifi <-> 1c22800.i2s mapping ok
[    7.632774] Adding 251464k swap on /dev/zram1.  Priority:5 extents:1 across:251464k SSFS
[    7.789616] zram0: detected capacity change from 0 to 52428800
[    8.175031] random: crng init done
[    8.175044] random: 7 urandom warning(s) missed due to ratelimiting
[    8.384240] EXT4-fs (zram0): mounted filesystem without journal. Opts: discard
[   15.137913] dwmac-sun8i 1c30000.ethernet eth0: PHY [0.1:01] driver [Generic PHY]
[   15.137934] dwmac-sun8i 1c30000.ethernet eth0: phy: setting supported 00,00000000,000062cf advertising 00,00000000,000062cf
[   15.141377] dwmac-sun8i 1c30000.ethernet eth0: No Safety Features support found
[   15.141395] dwmac-sun8i 1c30000.ethernet eth0: No MAC Management Counters available
[   15.141405] dwmac-sun8i 1c30000.ethernet eth0: PTP not supported by HW
[   15.141430] dwmac-sun8i 1c30000.ethernet eth0: configuring for phy/mii link mode
[   15.141454] dwmac-sun8i 1c30000.ethernet eth0: phylink_mac_config: mode=phy/mii/Unknown/Unknown adv=00,00000000,000062cf pause=10 link=0 an=1
[   15.146720] dwmac-sun8i 1c30000.ethernet eth0: phy link up mii/100Mbps/Full
[   15.146766] dwmac-sun8i 1c30000.ethernet eth0: phylink_mac_config: mode=phy/mii/100Mbps/Full adv=00,00000000,00000000 pause=0e link=1 an=0
[   15.146786] dwmac-sun8i 1c30000.ethernet eth0: Link is Up - 100Mbps/Full - flow control rx/tx
[   15.146812] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
[   33.782314] vcc3v0: disabling
[   33.782334] vcc5v0: disabling
[   33.782343] usb0-vbus: disabling
```

## :partly_sunny: Bosch Sensortec BME280 sensor information as table

```console
$ cat /proc/bme280info
I2C Adapter              : i2c-0
I2C Address              : 0x76

Chip Id                  : 0x60
Power Mode               : 0x0
Pressure Oversampling    : 0x5
Temperature Oversampling : 0x2
Humidity Oversampling    : 0x1
Filter Coefficient       : 0x4
Standby Time             : 0x0

Pressure                 : 75192
Temperature              : 2017
Humidity                 : 63985
```

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+end_of_line = lf
+insert_final_newline = true
+
+[*.sh]
+indent_style = space
+indent_size = 2
+
+[*.dts]
+indent_style = tab
+indent_size = 8
```
Comment (yekovalyov) [inline .editorconfig:14]: Is this file required for project?

**Comment by yekovalyov** [comment]
What is the purpose of 2 overlay files? Is one overlay file enough for this project?

File: .editorconfig (hunk 1 above)
Comment (malokhvii-eduard) [reply .editorconfig:14]: Yes. It helps maintain consistent coding styles across various editors and IDE.

**Comment by malokhvii-eduard** [comment]
- dt/sun8i-h3-bme280-overlay.dts - Custom overlay for BME280 sensor
-  dt/sun8i-h3-i2c0-overlay.dts - Auto-generated overlay after enabling i2c-0 via `armbian-config`
-  dt/sun8i-h3-orangepi-one.dts - Decompiled device tree for Orange Pi One
